src/webots/editor/WbProjectRelocationDialog.cpp

```diff
@@ -91,6 +91,7 @@ void WbProjectRelocationDialog::initCompleteRelocation() {
   mSourceEdit->setMinimumWidth(sourcePath.length() * 8);
 
   mTargetEdit = new WbLineEdit(mTargetPath, this);
+  mTargetEdit->setMinimumWidth(mTargetPath.length() * 8);
```

Comment:
I would be better this way:
```suggestion
#ifdef __APPLE__
  mTargetEdit->setMinimumWidth(mTargetPath.length() * 8);
#endif
```
So that we understand the problem is specific to macOS.